Core/PaymentMethodFactory.php

```diff
@@ -44,8 +50,11 @@ public static function create($sPaymentMethodType)
                 return new CreditCardPaymentMethod();
             case SofortPaymentMethod::getName(true):
                 return new SofortPaymentMethod();
+            case SepaCreditTransferPaymentMethod::getName(true):
```

Comment:
Same i find you need two getters, not one getter with a flag.

Reply:
not part of this pr

Reply:
Usually in are team if we see thinks to improve we just do it even if its not part of the PR. Its also a good practice, some link to read. -> https://www.matheus.ro/2017/12/11/clean-code-boy-scout-rule/

Reply:
as discussed we will refactor it in another ticket https://jira.parkside.at/browse/WDPD-140